### DISCUSSION TITLE
How to customize the path of Python? Couldn't launch python  exit code: 9009

### DISCUSSION THREAD

**Discussion by wzgrx** [vladmandic/sdnext#375] [Announcements]
### Issue Description

Couldn't launch python

exit code: 9009

**Comment by vladmandic** [comment]
This is issue with Python installation, not with core repo.
Btw, you have not provided ANY info, not even what OS you have or what Python do you have?
Transferring to discussions, hopefully someone can help there.

**Comment by wzgrx** [comment]
1. In webui.bat, the
@echo off
if not defined PYTHON (set PYTHON=""
)
if not defined VENV_ DIR (set "VENV_DIR=%~dp0%venv")
set up path
@echo off
if not defined PYTHON (set PYTHON="C:\UsersAppData\Local\Programs\Python\Python310\python.exe"
)
if not defined VENV_ DIR (set "VENV_DIR=%~dp0%venv")
2. Rerun bat
3. If No module named pip is displayed
4. Win+r cmd operation
py -m ensurepip
5. Rerun webui.bat

**Reply by dp23peace** [reply]
thank you!

**Reply by dp23peace** [reply]
Still didn't work for me, but thanks.  Tried to go back to an older version, 3.10, but then that wasn't available.  So, my options are to go to 3.12 beta, or use another program.   Super.
